docs/configuration/configuration.md

```diff
@@ -1501,6 +1505,69 @@ See [the Prometheus eureka-sd configuration file](/documentation/examples/promet
 for a practical example on how to set up your Eureka app and your Prometheus
 configuration.
 
+### `<scaleway_sd_config>`
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+#### Instance role
+
+* `__meta_scaleway_instance_id`: the id of the instance
+* `__meta_scaleway_instance_ip_is_public`: whether the ip of the target is publicly routed
+* `__meta_scaleway_instance_ip_version`: version of the ip `IPv4`, `IPv6`
+* `__meta_scaleway_instance_image_name`: name of the server image
+* `__meta_scaleway_instance_name`: name of the server
+* `__meta_scaleway_instance_status`: status of the server
+* `__meta_scaleway_instance_tags`: tag list of the server
+* `__meta_scaleway_instance_type`: commercial type of the server
+* `__meta_scaleway_instance_zone`: zone of the instance (`fr-par-1`, `fr-par-2`, `nl-ams-1`, `pl-waw-1`)
+
+#### Baremetal role
+
+* `__meta_scaleway_baremetal_id`: the id of the instance
+* `__meta_scaleway_baremetal_ip_is_public`: whether the ip of the target is publicly routed
+* `__meta_scaleway_baremetal_ip_version`: version of the ip `IPv4`, `IPv6`
+* `__meta_scaleway_baremetal_name`: name of the server
+* `__meta_scaleway_baremetal_os_name`: name of the os used
+* `__meta_scaleway_baremetal_os_version`: version of the os used
+* `__meta_scaleway_baremetal_status`: status of the server
+* `__meta_scaleway_baremetal_tags`: tag list of the server
+* `__meta_scaleway_baremetal_type`: commercial type of the server
+* `__meta_scaleway_baremetal_zone`: zone of the server (`fr-par-1`, `fr-par-2`, `nl-ams-1`, `pl-waw-1`)
+
+See below for the configuration options for Scaleway discovery:
+
+```yaml
+# The port to scrape metrics from.
+[ port: <int> | default = 80 ]
+
+# Refresh interval to re-read the targets list.
+[ refresh_interval: <duration> | default = 60s ]
+
+# API URL to use when doing the List requests
+[ api_url: <string> | default = "https://api.scaleway.com" ]
+
+# Secret key to use when listing servers. https://console.scaleway.com/project/credentials
+[ secret_key: <secret> ]
+
+# Project id to use when listing servers. https://console.scaleway.com/project/credentials
+[ project_id: <string> ]
+
+# Role can be either instance or baremetal
+[ role: <string> ]
+
+# Zone specify the availability zone of the server you target (fr-par-1, fr-par-2, nl-ams-1, pl-waw-1, ...)
```

Comment:
I would add an exemple and then a link to the API doc where you have the up to date list iof the zones.